### PR TITLE
Cut 0.1.11

### PR DESCRIPTION
**Pull request by kgang**
* Support newer pythons
* Moved tooluse back to tooluse dev dep group and placed imports inside of functions where the tooluse only modules are used
* Fix temperature default

### CODE REVIEW

File: concrete/clients.py [hunk 1]
```diff
@@ -29,10 +29,10 @@ class OpenAIClient(Client):
     Thin wrapper around open AI client
     """
 
-    def __init__(self, model: str | None = None, temperature: float | None = 0):
+    def __init__(self, model: str | None = None, temperature: float | None = None):
         OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
         self.client = OpenAI(api_key=OPENAI_API_KEY)
-        self.default_temperature = temperature or float(str(os.getenv("OPENAI_TEMPERATURE")))
+        self.default_temperature = temperature if temperature is not None else float(os.getenv("OPENAI_TEMPERATURE", 0))
```
Comment (MichaelDeng03) [inline concrete/clients.py:35]: nit: Does temperature or float(os.getenv("OPENAI_TEMPERATURE", 0)) not work?

**Review by MichaelDeng03 — APPROVED** [review]
looks good after poetry lock is fixed

File: concrete/clients.py (hunk 1 above)
Comment (kgang) [reply concrete/clients.py:35]: if temperature is 0, then it evaluates false

**Comment by kgang** [comment]
Dang I wonder if the build got messed up cuz poetry.lock is old

**Comment by kgang** [comment]
I'm gonna re-release as 0.1.11

**Pull request merged**